src/components/DownloadExperiment.js

```diff
@@ -0,0 +1,187 @@
+import React from 'react';
+
+import Button from './Button';
+import ProcessingGears from './ProcessingGears';
+import ModalManager from './Modal/ModalManager';
+import {
+  DatasetDownloadOptionsForm,
+  AggreationOptions,
+  TransformationOptions,
+  AdvancedOptions,
+  EmailAddressOptions,
+  SubmitDatasetOptionsForm,
+} from './DatasetDownloadOptionsForm';
+
+import { getDataSet } from '../api/dataSet';
+
+import { useLocalStorage } from '../common/hooks';
+import { getDomain } from '../common/helpers';
+
+const DownloadExperiment = ({ experiment, showProcessing = false }) => {
+  const [emailAddress] = useLocalStorage('email-address');
+
+  const {
+    accession_code: experimentAccessionCode,
+    num_downloadable_samples: downloadableSamplesCount,
+  } = experiment;
+
+  const data = {};
+  data[experimentAccessionCode] = experiment.samples.map(s => s.accession_code);
```

Comment:
It's also possible to initialize the property directly:

```suggestion
  const data = {
    [experimentAccessionCode]: experiment.samples.map(s => s.accession_code)
  };
```